apps/wallet-mobile/e2e/screens/prepareApp.screen.ts

```diff
@@ -0,0 +1,6 @@
+import { by, element } from 'detox'
+
+export const btn_SelectLanguageEnglish = () => element(by.id('languageSelect_en-US'))
+export const btn_Next = () => element(by.id('chooseLangButton'))
+export const chkbox_AcceptTos = () => element(by.id('acceptTosCheckbox'))
+export const btn_Accept = () => element(by.id('acceptTosButton'))
```

Comment:
The preferable way to name constants is <componentName><componentType>. The camelCase is preferable.
E.g. `selectLanguageEnglishButton` instead of `btn_SelectLanguageEnglish`

Reply:
It is just a custom way that I follow to increase the readability of the locator types (eg: btn, chkbox, lnk etc..).  If you think otherwise lmk and we can use the regular camel case.

Reply:
Okay. Let's keep it as it is currently. We will discuss it later.